docker/entrypoint.sh

```diff
@@ -16,7 +16,7 @@ if [ `id -u` -eq 0 ]; then
 fi
 
 if [ ! -d "${DRAND_HOME}/.drand" -a -n "${DRAND_PUBLIC_ADDRESS}" ]; then
-    drand generate-keypair --tls-disable "${DRAND_PUBLIC_ADDRESS}"
+    drand generate-keypair --insecure "${DRAND_PUBLIC_ADDRESS}"
```

Comment:
Do we really want the --insecure flag in the `generate-keypair` step here?

Reply:
I guess it says above "this file is used by the demo projects and CLI for building docker images", so it's not the docker entrypoint for our main daemon docker image ralease?

Reply:
hmm it does appear to be used in the real one here: https://github.com/drand/drand/blob/f8446f6a0ce361b66dd400eb0c0c19401e18611c/Dockerfile#L28; I guess in principle it should only get called if users haven't created their keys in advance, which they _should_ have done if they followed the instructions. Could certainly lead to unexpected problems though

Reply:
Then we need to remove the insecure I guess. But how comes it has been working so far in prod?